playbooks/libstatic.yml

```diff
@@ -0,0 +1,20 @@
+---
+# by default this playbook runs in the staging environment
+# to run in qa, pass '-e runtime_env=qa'
```

Comment:
There is no qa environment for libstatic.

Reply:
Thanks, @sandbergja! I removed that comment from the playbook.